### PR TITLE
Clean up gum_test.html and use getDisplayMedia().

### PR DESCRIPTION
**Pull request by jan-ivar**
@Pehrsons PTAL.

### CODE REVIEW

**Comment by jan-ivar** [comment]
[Whitespace diff](https://github.com/mozilla/webrtc-landing/pull/32/files?w=1).

File: gum_test.html [hunk 1]
```diff
@@ -85,146 +88,88 @@ <h4>getUserMedia is missing!</h4>
...
+  const video = document.createElement("video");
   video.setAttribute("width", 640);
   video.setAttribute("height", 480);
 
-  var muted = document.createElement("input");
+  const muted = document.createElement("input");
   muted.type = "checkbox";
   muted.checked = true;
-  muted.onclick = function() { video.muted = muted.checked; };
+  muted.onclick = () => { video.muted = muted.checked; };
```
Comment (Pehrsons) [inline gum_test.html:98]: We should change this to use track.enabled, or add additional controls for that.

**Review by Pehrsons — APPROVED** [review]
Great cleanup, thanks for doing this!

File: gum_test.html (hunk 1 above)
Comment (jan-ivar) [reply gum_test.html:98]: Notice I've renamed this to "☑ Volume mute" (aka "☑ Save ears") which is all this is.

I'll add `track.enabled` exercisers in a separate PR.

**Pull request merged**